lib/registry.js

```diff
@@ -96,7 +98,11 @@ class RegistryFetcher extends Fetcher {
         integrity: null,
       })
       const packument = await res.json()
-      packument._contentLength = +res.headers.get('content-length')
+      let contentLength = res.headers.get('content-length') ?? null
```

Comment:
Is this `??` effectively the same test as `isStream`? Do we really need them both?